moveit2/moveit2.repos

```diff
@@ -2,156 +2,195 @@ repositories:
   angles:
     type: git
     url: https://github.com/ros/angles.git
+    # branch: ros2
     version: 1.13.0
   backward_ros:
     type: git
     url: https://github.com/pal-robotics/backward_ros.git
-    version: foxy-devel
+    # branch: foxy-devel
+    version: 1.0.1
   control_msgs:
     type: git
     url: https://github.com/ros-controls/control_msgs.git
-    version: a555c37f1a3536bb452ea555c58fdd9344d87614
+    # branch: foxy-devel
+    version: 3.0.0
   control_toolbox:
     type: git
     url: https://github.com/ros-controls/control_toolbox.git
-    version: d7462e982e40bfdebddbcff8849c611caf8c8422
+    # branch: ros2-master
+    version: 2.0.2
   eigen_stl_containers:
     type: git
     url: https://github.com/ros/eigen_stl_containers.git
+    # branch: ros2
     version: 1.0.0
   geometric_shapes:
     type: git
-    url: https://github.com/ros-planning/geometric_shapes
+    url: https://github.com/ros-planning/geometric_shapes.git
+    # branch: ros2
     version: 2.1.2
   graph_msgs:
     type: git
     url: https://github.com/PickNikRobotics/graph_msgs.git
-    version: d0557ff5b19b3bdc9382e011c6613db29a69eb7b
-  ignition/ignition_cmake2_vendor:
+    # branch: ros2
+    version: 0.2.0
+  ignition_cmake2_vendor:
```

Comment:
Just curious why did you remove the `ignition` folder that these were in?

Reply:
I was comparing to the distribution.yaml for Rolling and noticed it didn't have the prefix. So, I removed it, trying to make it similar. However, it is present in the master branch of ros2.repos, so I will restore the prefix for those.

Reply:
`distribution.yaml` != a `.repos` file, I think the folder structure should be left there personally.

Reply:
Yes, I'll update to restore those as they were (with the 'ignition' prefix).

Reply:
Done.